src/chip-utils.h

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Move to src/engine/chipUtils.h (or just put this in dispatch.h maybe?)

Reply:
I moved this file into src/engine because https://github.com/tildearrow/furnace/blob/47574fdede7be644b65477a6eafabae8c29be930/src/gui/sysConf.cpp#L1576